CHANGELOG.md

```diff
@@ -1,6 +1,33 @@
 # Release Notes
 
-## Platform Version 0.10.1 - UNRELEASED
+## Platform Version 0.10.2 - UNRELEASED
+
+## Platform Version 0.10.1 - 2022-11-18
+
+* Record accumulator batches ([#2799](https://github.com/infinyon/fluvio/pull/2799))
```

Comment:
IMO we should mention here explicitly that some types were changed from i32 to u32

Reply:
To summarize, this PR replaces the internal value used for the `PartitionId`, `PartitionCount` and `ReplicationFactor` from `i32` to `u32` to avoid using `usize` which is arch based.

We can see it here: https://github.com/infinyon/fluvio/pull/2799/files#diff-0f8df827e6d2706f5c729e452d1db0a73a625cfb4d7e7450111f2f2468ee1a58R29

Will replace PR name with:

```
Replace `PartitionId`, `PartitionCount` and `ReplicationFactor` aliased type from `i32` to `u32`
``

And also improves performance of the RecordAccumulator `batches` method

Both entries points to the same PR

Reply:
Addressed on: 35f339e